test/index.test.ts

```diff
@@ -415,6 +419,201 @@ describe("Pipeline", () => {
   //   );
   // });
 
+  // it("search with promotions that have active pins with key fields that aren't in the values", async () => {
```

Comment:
Why commented out?

Reply:
All of these tests need adjusting to work without fetchMock and the different data structure.